86c71nd3q/feature/readme

Dolevg2411 opened the pull request.

Dolevg2411 committed:
Added readme to norman objects

AmitMizrachi618 committed:
Adding the licence file and updating README documentation to be short and concise.
Adding the licence file and updating README documentation to be short and concise.

(#86c71nd3q)

yonatankarimish commented on README.md:
```diff
@@ -1 +1,8 @@
+
+Norman Objects is a shared library that defines the core request and response models used across the Norman SDK and internal services.
+
+The library includes enums, data classes, and other structured objects that ensure consistency, type safety and clear contracts between services and SDK consumers.
```
The library includes classes, enums, and other structured objects that establish clear, consistent, type-safe contracts between backend services and SDK clients.

yonatankarimish commented on README.md:
```diff
@@ -1 +1,8 @@
+
+The library includes enums, data classes, and other structured objects that ensure consistency, type safety and clear contracts between services and SDK consumers.
+
+Comprehensive documentation is available on our website:
```
For the full reference and detailed instructions, please visit our Core SDK documentation at https://sdk.norman-ai.com/api/api-reference-overview/objects.

yonatankarimish commented on README.md:
```diff
@@ -1 +1,8 @@
\ No newline at end of file
+# Norman Objects Overview
+
+Norman Objects is a shared library that defines the core request and response models used across the Norman SDK and internal services.
```
Norman Objects is a shared library that defines the core request and response classes used across the Norman SDK and internal services.

(make sure not to overload the word "model", which we use for other purposes).

yonatankarimish reviewed (changes requested):
Some wording changes. Other than that looks good.

AmitMizrachi618 committed:
CR Wording fixes to objects readme documentation. small fixes.
CR Wording fixes to objects readme documentation. small fixes.

(#86c71nd3q)

AmitMizrachi618 committed:
Wording: Removing the word Overview from the headers, and the words core sdk from the bottom.
Wording: Removing the word Overview from the headers, and the words core sdk from the bottom.

(#86c71nd3q)

yonatankarimish reviewed (approved):
Approved
